### PR TITLE
Update User Location API markdown

### PR DESCRIPTION
**Pull request by KetanShrikhande**
Update from parent-fork

### CODE REVIEW

**Comment by charlieconcur** [comment]
Please bring this page into alignment with the [Documentation Template](https://github.com/concur/developer.concur.com/blob/preview/src/document-template.markdown).

**Comment by KetanShrikhande** [comment]
@charlieconcur - I am just modifying what was already approved and merged previously. This document was prepared by @brittanywest initially. I am curious as to why this approved and merged before if it did not align to the template?

**Comment by KetanShrikhande** [comment]
Ah of course I am one of the first to encounter the push...

**Comment by KetanShrikhande** [comment]
Template updated

File: src/api-reference/locate/userLocations.markdown [hunk 1]
```diff
@@ -35,6 +36,25 @@ This API supports POST only.
 3. Read the [Getting Started](/api-reference/authentication/getting-started.html) section of [Authentication API](/api-reference/authentication/apidoc.html).
 This API supports [Client Credentials Grant](/api-reference/authentication/apidoc.html#client_credentials) only. Your sandbox will be configured to accept posts from your application.
 
+
+## Service Details
+
+
+It is a traditional POST call to a REST endpoint which implements functionality by adhering to the following steps:
```
Comment (charlieconcur) [inline src/api-reference/locate/userLocations.markdown]: Our APIs aren't technically REST so remove this nomenclature.

**Comment by charlieconcur** [comment]
Thanks for updating to the template. We still have some formatting things to fix when you look at the whole page here in GitHub: https://github.com/concur/developer.concur.com/blob/bcbeb502d29b0af84c3525c022d5495413ea8fcc/src/api-reference/locate/userLocations.markdown

* Response Headers missing links to the actual RFCs.
* Status Codes missing links to the actual RFCs.
* Note: You can simply copy headers and status code from the template markup.
* The Request example is run on and uses cUrl. Separate data from request + headers.
* Tables appear to be missing data. Use a dash ("-") when there is no information provided rather than leaving blank.
* The last table is not rendering.
* Table markup still more complex than it needs to be.

**Comment by KetanShrikhande** [comment]
Thanks Charlie – I will work on the below points and update the PR

**Comment by charlieconcur** [comment]
I fixed a bunch of these issues for you -- so sync your branch / fork to see the changes.

**Comment by KetanShrikhande** [comment]
Thanks Charlie – much appreciated. Will have a look tomorrow morning first thing.

**Comment by KetanShrikhande** [comment]
fresh PR raised with all the comments incorporated

**Pull request closed without merging**